js/upload/src/Upload.svelte

```diff
@@ -129,11 +129,14 @@
 	async function loadFilesFromDrop(e: DragEvent): Promise<void> {
 		dragging = false;
 		if (!e.dataTransfer?.files) return;
-
 		const files_to_load = Array.from(e.dataTransfer.files).filter((f) => {
-			const file_extension =
-				f.type !== "" ? f.type : "." + f.name.split(".").pop();
-			if (file_extension && is_valid_mimetype(filetype, file_extension)) {
+			const f_type =
+				f.type !== "" ? f.type : "";
+			const f_extension = f.name !== "" ? "." + f.name.split(".").pop() : "";
```

Comment:
this can also be simplified:
```suggestion
      const f_extension = f.name.split(".").pop()
```